cime_config/machines/config_batch.xml

```diff
@@ -193,6 +193,33 @@
     </directives>
   </batch_system>
 
+  <batch_system type="pbspro" >
```

Comment:
Could you summarize how this section differs from the existing 'pbs' section above? Almost looks similar.

Reply:
there are slightly different in the command options, see the difference listed in this webpage, https://docs.alcf.anl.gov/running-jobs/pbs-qsub-options-table/

Reply:
It looks like lines 168-194 are the same as 196-221 (except commented out one). What am I missing?